src/styles/variables.scss

```diff
@@ -0,0 +1,4 @@
+$day-size: 100px;
+$gap: 1px;
+$start-days: (0 'mon', 1 'tue', 2 'wed', 3 'thu', 4 'fri', 5 'sat', 6 'sun');
```

Comment:
The '$start-days' variable is defined correctly, but ensure that it is used consistently throughout your SCSS files to dynamically apply the start-day modifiers.